clang/lib/Driver/SanitizerArgs.cpp

```diff
@@ -1137,6 +1137,23 @@ void SanitizerArgs::addArgs(const ToolChain &TC, const llvm::opt::ArgList &Args,
       return;
     GPUSanitize = true;
   }
+  // SPIR sanitizer support is experimental and will pass a fixed set of flags
+  if (TC.getTriple().isSPIR()) {
+    if (Sanitizers.has(SanitizerKind::Address)) {
+      CmdArgs.push_back(Args.MakeArgString("-fsanitize=address"));
```

Comment:
Since you don't change the arg string here in the runtime, you don't need to use rgs.MakeArgString, you can just push_back the literal string, like you did below.

Reply:
Thanks @yingcong-wu, will do